standard/docs/en/implementation/amendments.md

```diff
@@ -16,21 +16,43 @@ The nature of a change can be made explicit using:
 
 * **The amendments** building block. This can contain an array of amendment explanations, and clearly identify the releases that contain before and after values. 
 
-## Worked example
+## Worked examples
 
-In the example below:
+Each example includes sample JSON files located at the bottom of the page. Use the combobox at the end of the page to display the files for each example and step.
 
-* A tender is issued for a "Data merge tool" in a release with the `tag` 'tender'. 
-* This is updated in a release with the `tag` 'tenderUpdate' to provide an expanded description of the item. The publisher did not consider this to be a formal 'amendment' to the tender, so did not publish any amendment information.
-* The tender is then formally amended, with an increased value. This is published in a release with the `tag` 'tenderAmendment', and an `amendments` block under `tender`.
+### Example 1: Tender
 
+This example goes through updates during the **tender** stage in a contracting process.
+
+1. A tender is issued for a "Data merge tool" in a release with the `tag` 'tender'. 
+2. Information is updated in a new release with the `tag` 'tenderUpdate' to provide an expanded description of the item. The publisher did not consider this to be a formal 'amendment' to the tender, so did not publish any amendment information.
+3. Then there is a new update, where the value of the tender increases and the deadline for bid submissions is extended. This is considered as an 'amendment' by the publisher (depending of jurisdiction, both changes require an official approval), and so the new release has the `tag` 'tenderAmendment' and an `amendments` block under `tender`. The release reflects the updated value (USD 2000 instead of USD 1000) and the updated closing date for bid submissions (`2012-02-20` instead of `2012-02-15`).
```

Comment:
depending of jurisdiction, both changes require an official approval -> depending on jurisdiction, certain changes can need to be disclosed as amendment

Reply:
Outstanding typo:

'depending of jurisdiction' > 'depending on jurisdiction'